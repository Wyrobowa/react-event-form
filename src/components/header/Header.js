import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
    <div className="navbar-nav">
      <Link to="#" className="nav-item nav-link">Events list</Link>
      <Link to="#" className="nav-item nav-link">Add event</Link>
    </div>
  </nav>
);

export default Header;
