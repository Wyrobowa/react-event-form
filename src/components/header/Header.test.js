import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

it('renders InputFormField component without crashing', () => {
  shallow(<Header />);
});
