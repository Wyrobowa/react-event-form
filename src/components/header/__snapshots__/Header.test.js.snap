// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders Header component without crashing 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark bg-dark"
>
  <div
    className="navbar-nav"
  >
    <Link
      className="nav-item nav-link"
      to="/"
    >
      Attendies list
    </Link>
    <Link
      className="nav-item nav-link"
      to="/add"
    >
      Add attendee
    </Link>
  </div>
</nav>
`;
